shell2http.go

```diff
@@ -152,6 +155,14 @@ func getConfig() (cmdHandlers []Command, appConfig Config, err error) {
 		return nil, Config{}, fmt.Errorf("requires both -cert and -key options")
 	}
 
+	if len(os.Getenv(SH_BASIC_AUTH)) > 0 {
```

Comment:
Maybe better this variant?
```go
if basicAuth == "" && os.Getenv(SH_BASIC_AUTH) != "" {
    basicAuth = os.Getenv(SH_BASIC_AUTH)
}

if len(basicAuth) > 0 {
… and adding the original auth string to the error message is a good idea
        return nil, Config{}, fmt.Errorf("HTTP basic authentication from '%s' must be in format: name:password", basicAuth)
```

this will avoid duplication of code for parsing `login:pass` string